web-gateway/conf/application.conf

```diff
@@ -2,5 +2,9 @@
 
 lagom.circuit-breaker.default.call-timeout = 5s
 
+online-auction.instruction.show = false
+
 play.application.loader = loader.WebGatewayLoader
 play.filters.headers.contentSecurityPolicy = "img-src 'self' data:; default-src 'self'"
+
+items-search.page-size = 3
```

Comment:
This is pretty low, but I wanted the pagination features to be easily demonstrable. Let me know if you think another value is more appropriate.